class ValidationError(ValueError):
    """Custom exception for validation errors."""

    pass
